src/sysusers/sysusers.c

```diff
@@ -66,6 +66,7 @@ typedef struct Item {
         bool gid_set:1;
         bool gid_must_exist:1;
         bool uid_set:1;
+        bool force_reuse_id:1;
```

Comment:
Why do not you reuse `gid_must_exist` here?

Reply:
I am happy to do this. I was adding the new one to be more explicit, but i can also reuse the exisitng one, maybe rename to something like `gid_must_exist_force_reuse` along the way?

Happy to tweak my PR description if I'm unclear about the intent btw, just let me know what is unclear please :)

Reply:
Sorry for writing improper comment. I mean just I've not checked this in detail. I will look and test this later. Thank you.